bluepyopt/electrical/celltemplate.py

```diff
@@ -81,18 +80,19 @@ def create_empty_cell(name):
 
         # TODO minize hardcoded definition
         # E.g. sectionlist can be procedurally generated
-        template_content = 'begintemplate %s\n' \
-            'objref all, apical, basal, somatic, axonal\n' \
-            'proc init() {\n' \
-            'all 	= new SectionList()\n' \
-            'somatic = new SectionList()\n' \
-            'basal 	= new SectionList()\n' \
-            'apical 	= new SectionList()\n' \
-            'axonal 	= new SectionList()\n' \
-            'forall delete_section()\n' \
-            '}\n' \
-            'create soma[1], dend[1], apic[1], axon[1]\n' \
-            'endtemplate %s\n' % (name, name)
+        template_content = textwrap.dedent('''\
```

Comment:
bah, (name, name) instead of (name=name)

Not sure what is up with this patch, terrible.